main.js

```diff
@@ -0,0 +1,15 @@
+// Hamburger menu
+const menuIcon = document.querySelector('.menu-icon');
+const navMenu = document.querySelector('.nav-items');
+const navLinks = document.querySelectorAll('.nav-link');
+menuIcon.addEventListener('click', () => {
+  menuIcon.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+navLinks.forEach((navLink) => {
+  navLink.addEventListener('click', () => {
+    menuIcon.classList.remove('active');
+    navMenu.classList.remove('active');
+  });
+});
```

Comment:
- [ ] As mentioned in the project requirement you should make sure the menu follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins,...). Please kindly fix your design 🙃 😉 

Your design
<img width="363" alt="Screen Shot 2023-01-03 at 17 28 14" src="https://user-images.githubusercontent.com/74501523/210377227-5cc993ad-c4d5-4d0d-b51b-e57ad892c645.png">

Figma design
<img width="320" alt="Screen Shot 2023-01-03 at 17 29 23" src="https://user-images.githubusercontent.com/74501523/210377299-31c684b2-4e0d-486f-9b99-c0c649305153.png">